src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -93,15 +104,22 @@ export class ListingCreateComponent implements OnInit {
 
     ngOnInit() {
         this.store.dispatch(new JudgeActions.Get());
+        this.chosenListingType = ListingTypeTab.Single;
```

Comment:
Couldn't this be set at the top of the file?

Reply:
this way it will default to single session each time the component is opened.